openfisca_france_local/departements/eure_et_loir/pch.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+from openfisca_france.model.base import Variable, Individu, MONTH
```

Comment:
En complément d'une remarque plus bas dans ce fichier.
```suggestion
from openfisca_france.model.base import Variable, Individu, MONTH, not_
```